server/client_server.go

```diff
@@ -4,12 +4,20 @@
 package main
 
 import (
+	"fmt"
 	"net/http"
 
 	jira "github.com/andygrunwald/go-jira"
+	"github.com/hashicorp/go-version"
 	"github.com/pkg/errors"
 )
 
+const (
+	ServerInfoApiEndpoint = "rest/api/2/serverInfo"
+	CreateMetaAPIEndpoint = "rest/api/2/issue/createmeta/"
+	PivotVersion          = "8.4.0"
```

Comment:
What's significant about `8.4.0`? Isn't the actual pivot version `9.0.0`?

```suggestion
	JiraVersion9          = "9.0.0"
```

Reply:
@javaguirre The meaning of this constant is "The version of Jira that has a specific breaking change in their issues API". Maybe we name it something like `JiraVersionWithIssueAPIBreakingChange`, but something shorter hopefully.

Reply:
It seems off that one of these endpoints ends in a slash, and the other doesn't